src/MCPClient/lib/clientScripts/archivematicaTranscribeFile.py

```diff
@@ -8,18 +8,14 @@
 from tempfile import gettempdir
 from uuid import uuid4
 
-path = '/usr/share/archivematica/dashboard'
-if path not in sys.path:
-    sys.path.append(path)
+import django
+sys.path.append('/usr/share/archivematica/dashboard')
```

Comment:
We still run them as separate CLI scripts, so they're definitely being spun off as new processes at some level. IIRC our subprocess wrapper does support exporting the environment too.